astronomer/providers/amazon/aws/operators/sagemaker.py

```diff
@@ -0,0 +1,171 @@
+        :ref:``howto/operator:SageMakerTransformOperator``
+
+    :param config: The configuration necessary to start a transform job (templated).
+
+        If you need to create a SageMaker transform job based on an existed SageMaker model::
+
+            config = transform_config
+
+        If you need to create both SageMaker model and SageMaker Transform job::
+
+            config = {
+                'Model': model_config,
+                'Transform': transform_config
+            }
+
+        For details of the configuration parameter of transform_config see
+        :ref:``SageMaker.Client.create_transform_job``
+
+        For details of the configuration parameter of model_config, See:
+        :ref:``SageMaker.Client.create_model``
+
+    :param aws_conn_id: The AWS connection ID to use.
+    :param check_interval: If wait is set to True, the time interval, in seconds,
+        that this operation waits to check the status of the transform job.
+    :param max_ingestion_time: The operation fails if the transform job doesn't finish
+     within max_ingestion_time seconds. If you set this parameter to None, the operation does not timeout.
+    :param check_if_job_exists: If set to true, then the operator will check whether a transform job
+        already exists for the name in the config.
+    :param action_if_job_exists: Behaviour if the job name already exists. Possible options are "increment"
+        (default) and "fail".
+        This is only relevant if check_if_job_exists is True.
+    """
+
+    def execute(self, context: Context) -> None:  # type: ignore[override]
+        """
+        Creates transform job via sync hook `create_transform_job` and pass the
+        control to trigger and polls for the statis of the transform job in async
+        """
+        self.preprocess_config()
+        model_config = self.config.get("Model")
+        transform_config = self.config.get("Transform", self.config)
+        if self.check_if_job_exists:
+            self._check_if_transform_job_exists()
+        if model_config:
+            self.log.info("Creating SageMaker Model %s for transform job", model_config["ModelName"])
+            self.hook.create_model(model_config)
+        self.log.info("Creating SageMaker transform Job %s.", transform_config["TransformJobName"])
+        response = self.hook.create_transform_job(
+            transform_config,
+            wait_for_completion=False,
+        )
+        if response["ResponseMetadata"]["HTTPStatusCode"] != 200:
+            raise AirflowException(f"Sagemaker transform Job creation failed: {response}")
+        else:
+            end_time: Optional[float] = None
+            if self.max_ingestion_time is not None:
+                end_time = time.time() + self.max_ingestion_time
+            self.defer(
+                timeout=self.execution_timeout,
+                trigger=SagemakerTransformTrigger(
+                    poll_interval=self.check_interval,
```

Comment:
why we are using `check_interval` I think we decided to using poll_interval or poke_interval or I'm missing something

Reply:
It is inherited from `SageMakerTransformOperator` that uses `check_interval` for the polling mechanism we are using the same, in our code level we agreed to keep it `poke_interval`  now this is fixed